apps/web/src/components/dashboard/ExplorePossibleWidget/index.tsx

```diff
@@ -0,0 +1,240 @@
+    color: string
+  }
+}
+
+const ITEM_WIDTH = 180
+const ITEM_GAP = 16
+
+const ExplorePossibleWidget = () => {
+  const router = useRouter()
+  const txBuilderApp = useTxBuilderApp()
+  const isDarkMode = useDarkMode()
+  const [canScrollLeft, setCanScrollLeft] = useState(false)
+  const [canScrollRight, setCanScrollRight] = useState(false)
+  const scrollContainerRef = useRef<HTMLUListElement>(null)
+
+  const EXPLORE_POSSIBLE_APPS: ExplorePossibleApp[] = useMemo(
+    () => [
+      {
+        id: 'swap',
+        title: 'Swap tokens instantly',
+        iconUrl: isDarkMode
+          ? '/images/explore-possible/swap-large-dark.svg'
+          : '/images/explore-possible/swap-large.svg',
+        link: {
+          pathname: AppRoutes.swap,
+          query: { safe: router.query.safe },
+        },
+        tag: {
+          text: 'Popular',
+          color: '#12FF80',
+        },
+      },
+      {
+        id: 'spaces',
+        title: 'Manage multiple Safes',
+        iconUrl: isDarkMode
+          ? '/images/explore-possible/spaces-large-dark.svg'
+          : '/images/explore-possible/spaces-large.svg',
+        link: 'https://app.safe.global/welcome/spaces',
+      },
+      {
+        id: 'transaction-builder',
+        title: 'Build custom transaction',
+        iconUrl: isDarkMode
+          ? '/images/explore-possible/tx-builder-large-dark.svg'
+          : '/images/explore-possible/tx-builder-large.svg',
+        link: txBuilderApp?.link || {
+          pathname: AppRoutes.apps.index,
+          query: { safe: router.query.safe },
+        },
+      },
+      {
+        id: 'walletconnect',
+        title: 'Connect to Web3 apps',
+        iconUrl: isDarkMode
+          ? '/images/explore-possible/apps-large-dark.svg'
+          : '/images/explore-possible/apps-large.svg',
+        link: {
+          pathname: AppRoutes.apps.index,
+          query: { safe: router.query.safe },
+        },
+      },
+    ],
+    [router.query.safe, txBuilderApp, isDarkMode],
+  )
+
+  const updateScrollState = () => {
+    const container = scrollContainerRef.current
+    if (!container) return
+
+    const newCanScrollLeft = container.scrollLeft > 0
+    const newCanScrollRight = container.scrollLeft < container.scrollWidth - container.clientWidth - 1
+
+    setCanScrollLeft(newCanScrollLeft)
+    setCanScrollRight(newCanScrollRight)
+  }
+
+  useEffect(() => {
+    updateScrollState()
+    window.addEventListener('resize', updateScrollState)
+    return () => window.removeEventListener('resize', updateScrollState)
+  }, [])
+
+  const scrollList = (direction: 'left' | 'right') => {
+    const container = scrollContainerRef.current
+    if (!container) return
+
+    const scrollAmount = ITEM_WIDTH + ITEM_GAP
+    const newScrollLeft =
+      direction === 'left' ? container.scrollLeft - scrollAmount : container.scrollLeft + scrollAmount
+
+    container.scrollTo({
+      left: newScrollLeft,
+      behavior: 'smooth',
+    })
+  }
```

Comment:
Please check if this can be done purely in HTML/CSS. There's a native CSS property for this: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-type